hoomd/md/EvaluatorPairDipole.h

```diff
@@ -187,19 +225,19 @@ class EvaluatorPairDipole
             Scalar r5inv = r3inv*r2inv;
 
             // convert dipole vector in the body frame of each particle to space frame
-            vec3<Scalar> p_i = rotate(quat<Scalar>(quat_i), vec3<Scalar>(params.mu, 0, 0));
-            vec3<Scalar> p_j = rotate(quat<Scalar>(quat_j), vec3<Scalar>(params.mu, 0, 0));
+            vec3<Scalar> p_i = rotate(quat<Scalar>(quat_i), vec3<Scalar>(mu, 0, 0));
```

Comment:
We could leave this for a later PR, but I'd like to generalize the potential to allow vector mu's per particle. We can use the `shape` data to store a vector mu per particle type, but expose that to the user as a tuple type parameter:
```
dipole.mu['A'] = (x,y,z)
```

Reply:
Done.